external/downstream/cct3/CMakeLists.txt

```diff
@@ -16,8 +16,7 @@ if(${ENABLE_cct3})
 
         ExternalProject_Add(cct3_external
             DEPENDS psi4-core
-            #URL https://github.com/piecuch-group/cct3/archive/master.tar.gz
-            URL https://github.com/loriab/psi4_cct3/archive/cmake.tar.gz
+            URL https://github.com/piecuch-group/cct3/archive/master.tar.gz
```

Comment:
Is there a tagged version we could get instead of master?  Just for futureproofing

Reply:
good point (especially as they use `dev`, not `master`, as main branch). cct3 is going into long-term stable. also, they have no tags, and I'm waiting to ask for one until after I build the conda package, but I put in a hash.